test/fossa_travis_tests.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Added tests so that in the future, we can hook up Travis_CI to "check itself" and prove the scripts will work for clients.